health-services/PassiveGoalsCompose/gradle/libs.versions.toml

```diff
@@ -1,42 +1,41 @@
 [versions]
-accompanist-permissions = "0.36.0"
-android-gradle-plugin = "8.7.0"
-androidx-activity = "1.9.2"
-androidx-datastore = "1.1.1"
-androidx-lifecycle = "2.8.6"
-androidx-wear-compose = "1.4.0"
-androidx-wear-tiles = "1.0.1"
-androidx-wear-watchface = "1.0.1"
-androidx-work = "2.9.1"
-compose = "1.7.3"
+accompanist-permissions = "0.37.0"
+android-gradle-plugin = "8.7.3"
+androidx-activity = "1.10.1"
+androidx-datastore = "1.1.7"
+androidx-lifecycle = "2.9.0"
+androidx-wear-compose = "1.4.1"
+androidx-work = "2.10.1"
+compose = "1.7.8"
+compose-tooling = "1.8.2"
 compose-compiler = "1.5.15"
 concurrent-futures = "1.2.0"
 guava = "33.3.1-android"
-health-services = "1.0.0-rc02"
+health-services = "1.0.0"
 ktlint = "0.46.1"
 org-jetbrains-kotlin = "1.9.25"
-org-jetbrains-kotlinx = "1.6.0"
+wear-compose-material3 = "1.5.0-beta02"
+wear-ui-tooling-version = "1.0.0"
+horologist = "0.7.5-alpha"
```

Comment:
there should be a Horologist beta available, can you update to the latest?

Reply:
is `0.7.14-beta` the one?